layers/meta-resin-iot2000/conf/samples/bblayers.conf.sample

```diff
@@ -0,0 +1,28 @@
+# LAYER_CONF_VERSION is increased each time build/conf/bblayers.conf
+# changes incompatibly
+LCONF_VERSION = "6"
+
+BBPATH = "${TOPDIR}"
+BBFILES ?= ""
+
+BBLAYERS ?= " \
+    ${TOPDIR}/../layers/poky/meta \
+    ${TOPDIR}/../layers/poky/meta-yocto \
+    ${TOPDIR}/../layers/meta-openembedded/meta-oe \
+    ${TOPDIR}/../layers/meta-openembedded/meta-filesystems \
+    ${TOPDIR}/../layers/meta-openembedded/meta-networking \
+    ${TOPDIR}/../layers/meta-openembedded/meta-python \
+    ${TOPDIR}/../layers/meta-resin/meta-resin-common \
+    ${TOPDIR}/../layers/meta-resin/meta-resin-morty \
+    ${TOPDIR}/../layers/meta-intel \
+    ${TOPDIR}/../layers/meta-resin-iot2000 \
+    ${TOPDIR}/../layers/meta-iot2000/meta-iot2000-bsp \
+    ${TOPDIR}/../layers/oe-meta-go \
```

Comment:
Please also include meta-rust. Like this board does: https://github.com/resin-os/resin-artik/blob/master/layers/meta-resin-artik/conf/samples/bblayers.conf.sample#L19